Report_templates/stagehydrograph.Rmd

```diff
@@ -18,15 +18,16 @@ wy_start <- wy_dates$wy_start
 wy_end <- wy_dates$wy_end
 
 isDevilsLake <- siteNumber %in% c('05404500')
+startDate <- ifelse(test = isDevilsLake, 
+                      yes = as.Date("1936-01-01"), 
+                      no = as.Date("1800-01-01"))
```

Comment:
this is where we should remove `as.Date`. I think it should just be `yes = "1936-01-01"` and `no = "1800-01-01"`